<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="6" hostname="DESKTOP-K5QR6PC" failures="3" tests="7" name="Test" time="156.61" errors="0" timestamp="2022-08-10T13:04:55 IST">
  <testcase classname="com.mystore.testcases.TC_SearchResultPage" name="validateProductAvailability" time="14.091"/>
  <testcase classname="com.mystore.testcases.TC_HomePage" name="orderHistoryTest" time="21.184"/>
  <testcase classname="com.mystore.testcases.TC_LoginPage" name="validateLogin" time="19.056">
    <failure type="java.lang.AssertionError" message="expected [http://automationpractice.com/index.php?controller=my-account] but found [http://automationpractice.com/index.php?controller=my-account123]">
      <![CDATA[java.lang.AssertionError: expected [http://automationpractice.com/index.php?controller=my-account] but found [http://automationpractice.com/index.php?controller=my-account123]
at com.mystore.testcases.TC_LoginPage.validateLogin(TC_LoginPage.java:35)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- validateLogin -->
  <testcase classname="com.mystore.testcases.TC_AddToCartPage" name="addToCartTest" time="24.18"/>
  <testcase classname="com.mystore.testcases.TC_HomePage" name="wishListTest" time="12.433"/>
  <testcase classname="com.mystore.testcases.TC_SearchResultPage" name="validateProductAvailability" time="36.047">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.mystore.testcases.TC_SearchResultPage.validateProductAvailability(TC_SearchResultPage.java:28)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- validateProductAvailability -->
  <testcase classname="com.mystore.testcases.TC_SearchResultPage" name="validateProductAvailability" time="32.162">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.mystore.testcases.TC_SearchResultPage.validateProductAvailability(TC_SearchResultPage.java:28)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- validateProductAvailability -->
  <testcase classname="com.mystore.base.Base" name="@AfterSuite afterSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite afterSuite -->
  <testcase classname="com.mystore.base.Base" name="@AfterSuite afterSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite afterSuite -->
  <testcase classname="com.mystore.base.Base" name="@AfterSuite afterSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite afterSuite -->
  <testcase classname="com.mystore.base.Base" name="@BeforeSuite loadConfig" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite loadConfig -->
  <testcase classname="com.mystore.base.Base" name="@BeforeSuite loadConfig" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite loadConfig -->
  <testcase classname="com.mystore.base.Base" name="@BeforeSuite loadConfig" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite loadConfig -->
</testsuite> <!-- Test -->
